### ISSUE TITLE
Coding style should be fixed

### ISSUE THREAD

**Issue by michael-e** [symphonycms/members#302]
We should fix the extension's coding style (and make it PSR-2 comaptible). However, I suggest to wait until most of the other issues (and pull requests!) are closed.

**Comment by nitriques** [comment]
Do you have experience with automated tools ? I've tried `phpcs` a couple of time, but still need practice with it.

**Comment by michael-e** [comment]
No, unfortunately not. I prefer Sublime Text packages, because I want to check the result anyway. There are two or three PHPCS-related packages, e.g. https://github.com/adael/SublimePhpCsFixer (You will have to install php-cs-fixer globally for this package to work — `composer global require friendsofphp/php-cs-fixer`.)

**Comment by nitriques** [comment]
Ok thanks!
